docs/open-api.yaml

```diff
@@ -0,0 +1,185 @@
+openapi: 3.0.3
```

Comment:
We use "openapi.yaml" in other projects, so please use same name convention in filename